### DISCUSSION TITLE
Backlight   and   (pwm tone/no_tone) conflict.

### DISCUSSION THREAD

**Discussion by flexiti** [HASwitchPlate/openHASP#708] [Q&A]
When using audio (pwm)  generation from the port (In my case, port 1), the backlight turns off and you need to use ` haspDevice.set_backlight_pin(38); `to restore the backlight. 
It's as if the backlight port setting was lost. Anyone had a problem with this?

I change  `#define BACKLIGHT_CHANNEL 1 // pwm channel 0-15` to different  but it not helps

**Comment by fvanroie** [comment]
This sounds like a hardware limitation of either the device or the ESP32 chip. Without knowing more about the hardware, it's anyone's guess.
